tests/queryset/transform.py

```diff
@@ -197,5 +197,17 @@ class Location(Document):
         update = transform.update(Location, set__poly={"type": "Polygon", "coordinates": [[[40, 5], [40, 6], [41, 6], [40, 5]]]})
         self.assertEqual(update, {'$set': {'poly': {"type": "Polygon", "coordinates": [[[40, 5], [40, 6], [41, 6], [40, 5]]]}}})
 
+    def test_type(self):
```

Comment:
It would make sense to add a docstring here linking to http://docs.mongodb.org/manual/reference/operator/query/type/. At a first glance, the 1/8/2/16 is a mystery in this test.